README.md

```diff
@@ -183,20 +206,67 @@ function Child({ onChangeColor, color }) {
 }
 ```
 
+Uh-oh, TypeScript's warning us again! Because we added a new prop, we need to
+type it as well so our component knows exactly what to expect. Let's add it to
+our interface. This time, we're not passing down a function, the `color` prop is
+just passing down a string:
+
+```ts
+// Child.tsx
+interface Props {
+  onChangeColor(): void;
+  color: string;
+}
+```
+
 Lastly, we have to update the `handleChangeColor()` function in `Parent` to
 change not just the `color` state, but also the `childrenColor`. To practice
 sending data _back_ to the parent, let's change our `handleChangeColor` to take
 in an argument of `newChildColor` and then use that variable to update the state
 of the `Child` component:
 
 ```jsx
+// Parent.tsx
 function handleChangeColor(newChildColor) {
   const newRandomColor = getRandomColor();
   setColor(newRandomColor);
   setChildrenColor(newChildColor);
 }
 ```
 
+Oh no! TypeScript is warning us of something again. It says:
+
+`TS2322: Type '(newChildColor: any) => void' is not assignable to type '() => void'.`
+
+on the line:
+
+`<Child color={childrenColor} onChangeColor={handleChangeColor} />`
+
+This is because we've now added a parameter to the `handleChangeColor` function
```

Comment:
Our convention with JS is to include the `()` in function names. It gets missed all the time, but figured I'd mention it...